packages/tupaia-web/src/features/Dashboard/Dashboard.tsx

```diff
@@ -53,9 +53,8 @@ const ScrollBody = styled.div`
   position: relative;
   height: 100%;
   @media screen and (min-width: ${MOBILE_BREAKPOINT}) {
-    overflow: auto;
-    // Fix issue where scroll bar appears and dis-appears repeatedly and flickers
-    margin-right: 3px;
+    // Need to be permanently set to scroll to fix issue where scroll bar appears and dis-appears repeatedly and flickers
```

Comment:
My only question around this would be, if there are cases where scroll isn't actually needed, e.g. 1 dashboard item only, then the scroll would still show right? Would be nicer if we didn't need to do that, I think